internal/testutil/acc/attribute_checks.go

```diff
@@ -54,3 +54,23 @@ func JSONEquals(expected string) resource.CheckResourceAttrWithFunc {
 		return nil
 	}
 }
+
+func AddAttrSetChecks(targetName string, checks []resource.TestCheckFunc, attrNames ...string) []resource.TestCheckFunc {
```

Comment:
Hmm. It comes from the underlying `resource.TestCheckResourceAttrSet` from the `terraform-plugin-testing`.
I thought it would be smart to keep the names the same. The relevant calls are:

1. `acc.AddAttrSetChecks(datasourceName, checks, "project_id", "index_id")`
2. `acc.AddAttrSetChecks(resourceName, checks, "project_id")`

WDYT?